src/components/OverviewApp/Notary.tsx

```diff
@@ -558,6 +564,63 @@ const Notary = (props: { notaryProps: NotaryProps }) => {
       }
    };
 
+   const signRemovalRequest = async () => {
+      try {
+         setRemovalLoading(true);
+         const dataCapBytes : number= anyToBytes(removeDataCapIssue?.datacapToRemove as string)
+         console.log("sign removal request")
+         const idAddress = await context.wallet.api.actorAddress(removeDataCapIssue?.address);
+         const message = {
+            VerifiedClient: idAddress,
+            DataCapAmount: dataCapBytes,
+            RemovalProposalID: 1234
+         }
+
+         const encodedMessage = context.wallet.api.encodeRemoveDataCapParameters(
+            [message.VerifiedClient, message.DataCapAmount, [message.RemovalProposalID]]
```

Comment:
The encoded message is missing this string `SignatureDomainSeparation_RemoveDataCap` (see https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/verifreg/verified_registry_state.go#L97)
This should be added in the api.encodeRemoveDataCapParameters function.